cmd/execution_builder.go

```diff
@@ -220,7 +220,6 @@ func (builder *ExecutionNodeBuilder) LoadComponentsAndModules() {
 		// TODO: will re-visit this once storehouse has implemented new WAL for checkpoint file of
 		// payloadless trie.
 		// Component("execution data pruner", exeNode.LoadExecutionDataPruner).
-		Component("observer collection indexer", exeNode.LoadObserverCollectionIndexer).
```

Comment:
The same component is initialized in L239, this is to remove a duplicated component.